app/src/components/inputs/inputDropdown/inputDropdown.jsx

```diff
@@ -78,14 +80,15 @@ export class InputDropdown extends Component {
         active={option.active}
         text={option.text}
         multiple={this.props.multiple}
+        onChange={this.props.onChange}
       />
     ));
   }
 
   render() {
     const classes = cx({
       dropdown: true,
-      opened: this.props.active,
+      opened: this.state.opened,
     });
     return (
       <div ref={(node) => { this.node = node; }} className={classes}>
```

Comment:
Define classes directly here without const. Please fix condition rendering in that component: multiple &&